examples/cord_epsim/main.c

```diff
@@ -67,33 +69,54 @@ static gcoap_listener_t listener = {
 
 int main(void)
 {
+    char addrstr[IPV6_ADDR_MAX_STR_LEN];
     puts("Simplified CoRE RD registration example\n");
 
     /* fill riot info */
     sprintf(riot_info, "{\"ep\":\"%s\",\"lt\":%u}",
             cord_common_get_ep(), CORD_LT);
 
     /* parse RD address information */
-    sock_udp_ep_t rd_ep = {
-        .family    = AF_INET6,
-        .netif     = SOCK_ADDR_ANY_NETIF,
-        .port      = RD_PORT,
-    };
-
-    /* parse RD server address */
-    if (ipv6_addr_from_str((ipv6_addr_t *)&rd_ep.addr.ipv6, RD_ADDR) == NULL) {
+    sock_udp_ep_t rd_ep;
+    rd_ep.family = AF_INET6;
+    rd_ep.netif = SOCK_ADDR_ANY_NETIF;
+    rd_ep.port = 0;
```

Comment:
```suggestion
```

`sock_udp_str2ep()` is doing a memset to 0.

Reply:
Right